app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/webrtc/server/WebRtcManager.kt

```diff
@@ -194,12 +218,20 @@ class WebRtcManager constructor(
 
     fun getConnectionObservable(): Observable<RtcConnectionState> =
         connectionObserver.streamObservable.ofType(ConnectionState::class.java)
+            .doOnNext {
+                if (it.connectionState is RtcConnectionState.Disconnected
+                    || it.connectionState is RtcConnectionState.Error
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><table>
  <tbody>
    <tr>
      <td><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/9676b6813419b87b660bdcb0fa74aec9360fe789/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/webrtc/server/WebRtcManager.kt#L223"></span>Line must not begin with “</td>
      <td> </td>
      <td>”</td>
    </tr>
  </tbody>
</table>
</td>
    </tr>
  </tbody>
</table>